src/ObjectVisualisation/Item/MetricImage.tsx

```diff
@@ -11,27 +11,44 @@ type Props = {
 
 export const MetricImage = ({ position, itemWidth, item, itemSelectHandler }: Props) => {
   const [image, setImage] = useState(new window.Image());
+  const [isMousedOver, setIsMousedOver] = useState(false);
 
   useEffect(() => {
     const temp = new window.Image();
-    temp.src = 'https://raw.githubusercontent.com/fylip97/Thesis/main/src/img/metrics.png';
+    if (!isMousedOver) {
+      temp.src = 'https://raw.githubusercontent.com/fylip97/Thesis/main/src/img/metrics.png';
```

Comment:
Can we maybe store the images in a different place than in the repo of @fylip97 ?
:)

Reply:
yes definitely :) i found two unused images in the repo so i tried to use them :) but i can store them inside the directory or link them from an external https link.

Reply:
Ok cool :) I'd prefer to store them inside the directory, such that the images are under version control and newest image is always used.

Reply:
Tobi i just had a small thought: when i reference the image using the relative path like "../../src/img/metrics.png" or "c:/program files..." ,wouldn't that potentially cause a bug trying to compile/build the plugin on a different OS like Linux or mac (because the file system is different)?

Reply:
You should be able to define the path in the typical HTML manner. Afaik, the [HtmlImageElement](https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_dom_d_.htmlimageelement.html) only transforms the given path into an HTML image tag.

IMO it would be sufficient to create a new folder `src/img`.

You can then reference the image as follows:
`temp.src = 'img/metrics.png'`